app/src/main/java/org/fossasia/openevent/app/main/MainActivity.java

```diff
@@ -114,7 +115,7 @@ public void onDrawerClosed(View drawerView) {
 
                 if (id == R.id.nav_logout)
                     showLogoutDialog();
-                else if (eventId != -1)
+                else if (eventId != -1 || id == R.id.nav_settings)
```

Comment:
this case is not required now, as we are hiding event specific options already if event id is not stored.